templates/index.html

```diff
@@ -3,55 +3,165 @@
-ballots have been requested for the January special election and {{
-stats.returned_special|commafy }} have been returned. There were {{
-stats.returned_general|commafy }} ballots returned early in the general
-election, so we are
-{{ (100.0 * stats.returned_special / stats.returned_general)|round(1) }}% of the way there.
-
-<div class="container py-3">
-  <form class="card p-3 p-lg-4" method="GET" action="/search">
-    <h2 class="h4">Search by name</h2>
-    <p><em>Don't use middle initials, suffixes like &ldquo;Sr,&rdquo; etc.</em></p>
-    <div class="form-row">
-      <div class="col-md-6 col-sm-12 mb-3">
-        <label for="input-first">First / given name</label>
-        <input type="text" class="form-control" placeholder="Given name" name="first" id="input-first" required aria-required="true">
-      </div>
-      <div class="col-md-6 col-sm-12 mb-3">
-        <label for="input-last">Last / family name</label>
-        <input type="text" class="form-control" placeholder="Family name" name="last" id="input-last" required aria-required="true">
+          <!-- TODO: Decide which p copy to keep...or tweak copy. And get stats values right? -->
+          <!-- <p class="card-text">And we're behind: by this point in the November election,
+            <strong>{{ stats.returned_general|commafy }} voters</strong> had already voted. But only
+            <strong>{{ stats.returned_special|commafy }} voters</strong> have voted so far in the
+            runoff. Help us catch up!</p> -->
+
+          <p class="card-text mb-0">
+            So far, <strong>{{ stats.applied_special|commafy }} ballots</strong> have been requested
+            for the January special election and <strong>{{
+          stats.returned_special|commafy }} ballots</strong> have been returned. There were
+            <strong>{{
+          stats.returned_general|commafy }} ballots </strong> returned early in the general
+            election, so we are only
+            <strong>{{ (100.0 * stats.returned_special / stats.returned_general)|round(1) }}%</strong>
+            of the way there
+            <em>(as of {{ stats.update_time.strftime('%B %-d at %-I:%M %p EST') }})</em>.
+          </p>
       </div>
     </div>
-    <div class="form-row">
-      <div class="col-md-12 col-sm-12 mb-3">
-        Optional fields:
-      </div>
+
+    <div>
+      <h3 class='text-center'>Do your part: Help your friends in Georgia vote</h3>
+      <hr class='my-5'>
     </div>
-    <div class="form-row">
-      <div class="col-md-4 col-sm-12 mb-3">
-        <label for="input-first">Middle name/initial</label>
-        <input type="text" class="form-control" placeholder="Middle name" name="middle" id="input-middle">
+
+    <div>
+      <!-- TODO: Have the # be its own column and everything else to the right for a bit more separation? Depends if we are cool with the wider container though. -->
```

Comment:
If we go wider, could make more of a separation... or make it happen here as-is in the 640px. Weird that `.container-md` jumps to 1140px